### PR TITLE
Add alert system & Patches

### PR DESCRIPTION
**Pull request by zelytra**

### CODE REVIEW

File: frontend/src/vue/alert/AlertUtils.ts
Comment (dadodasyra) [inline frontend/src/vue/alert/AlertUtils.ts:1]: Why hardcode messages (in french !)

**Review by dadodasyra — CHANGES_REQUESTED** [review]
No idea how to handle error from rust..

File: frontend/src/vue/alert/AlertUtils.ts
Comment (zelytra) [reply frontend/src/vue/alert/AlertUtils.ts:1]: My bad this is a component i use in a lot of my app. I take care of this

**Review by dadodasyra — APPROVED** [review]
Made some comments on config page which contain same changes

**Pull request merged**